dist/ba_data/python/babase/_meta.py

```diff
@@ -327,7 +327,7 @@ def _scan_module(self, moduledir: Path, subpath: Path) -> None:
         meta_lines = {
             lnum: l[1:].split()
             for lnum, l in enumerate(flines)
-            if '# ba_meta ' in l
```

Comment:
Is this a change in the official game API?

Reply:
> Is this a change in the official game API?

Made a PR in the official ballistica repo too, will be pulled soon ig.

Reply:
Then, it is not required to explicitly raise PR here for this change, it will be pulled with other changes from ballisitca repo.
Taking partial change from that repo will later add merge conflicts here and chance to break some functionality. 
Such changed not required unless its a bug fix or a severe security issue.